### DISCUSSION TITLE
Allow Users without admin access to download subtitles

### DISCUSSION THREAD

**Discussion by gotton** [jellyfin/jellyfin#5387] [General] [answered]
Hello,

I'm sharing my jellyfin with the family over the internet.

But they are not admin and there access is restricted because i don't want them to mess with the config or so.

But only the admin have the ability to download subtitles. It's quite annoying as i dont have download all subtitles. So they have to ask me to download the subtitles via opensubtitles plugin then they could choose it.

Is there a way that non admin people could have access to the option "change subtitles" when accessing jellyfin ?

**Comment by anthonylavado** [comment, marked as answer]
Not yet, but it is something we want to work on!

See the Feature Request here: https://features.jellyfin.org/posts/185/non-admin-users-download-subtitles

**Reply by gotton** [reply]
Ok. Sorry i didn't saw this post.

thanks for your answer
